packages/hs-react-ui/src/components/RangeSlider/RangeSlider.tsx

```diff
@@ -156,13 +159,43 @@ export const DomainLabel = styled.div`
   }}
 `;
 
-export default ({
+export const Marker = styled(Div)`
+  ${({ sliderPosition = 0 }) => {
+    const { colors } = useTheme();
+    return `
+      position: absolute;
+      text-align: center;
+      display: flex;
+      justify-content: center;
+      height: 1rem;
+      width: 2px;
+      left: ${sliderPosition}px;
+      background-color: ${colors.grayLight};
+    `;
+  }}
+`;
+export const MarkerLabel = styled(Div)`
+  ${({ color }) => {
+    const { colors } = useTheme();
+    return `
+    position: absolute;
+    bottom: 100%;
+    white-space: nowrap;
+    font-size: .375rem;
+    color: ${color || colors.grayLight};
+  `;
+  }}
+`;
+
+export const RangeSlider = ({
   StyledContainer = Container,
   StyledDragHandle = DragHandle,
   StyledHandleLabel = HandleLabel,
   StyledSlideRail = SlideRail,
   StyledSelectedRangeRail = SelectedRangeRail,
   StyledDomainLabel = DomainLabel,
+  StyledMarker = Marker,
+  StyledMarkerLabel = MarkerLabel,
```

Comment:
These should get markerProps and markerLabelProps as well

Reply:
great catch!